usaspending_api/api_contracts/contracts/award/idv/FundingAccounts.md

```diff
@@ -0,0 +1,46 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Funding Treemap
+
+These endpoints are used to power USAspending.gov's IDV Summary Funding Treemap component.
+
+## List Federal Accounts [/api/v2/awards/idvs/accounts/]
+
+This endpoint returns a list of federal accounts under a given IDV.
+
+
+### List Federal Accounts [POST]
++ Request (application/json)
+    + Attributes
+        + `award_id`:`CONT_AW_4732_-NONE-_GS30FHA006_-NONE-` (required, string) 
+            IDV to return accounts for
+        + `page`: 1 (optional, number)
+            Page number to return
+        + `limit`: 10 (optional, number)
+            Maximum number to return
+        + `order`: `desc` (optional, string)
+            Direction of sort
+        + `sort`:`total_transaction_obligated_amount` (optional, string)
+            The field to sort on
+
++ Response 200 (application/json)
+    + Attributes 
+       + page_metadata: (required, PageMetadata)
+       + results: (required, array[AccountListing])
```

Comment:
Can we get rid of the `:` after `page_metadata` and `results`?  They throw unnecessary warnings in Dredd.